bundle/terranova/apply.go

```diff
@@ -33,9 +33,10 @@ func (d *DeploymentUnit) Destroy(ctx context.Context, db *tnstate.TerranovaState
 }
 
 func (d *DeploymentUnit) Deploy(ctx context.Context, db *tnstate.TerranovaState, inputConfig any, actionType deployplan.ActionType) error {
+	// Note, config may be different between plan and deploy due to resolved $resource references
```

Comment:
Do you say this to justify the "PrepareConfig" call, or as a general reminder?

Reply:
Yes, this explains why PrepareConfig call is not cached between plan and apply.

Reply:
> I like that idea. More explicit.

Noted for follow up, since PrepareConfig was not added here.